diff --git a/node_modules/antd/es/button/button.js b/node_modules/antd/es/button/button.js
index f84bca7..cb8384c 100644
--- a/node_modules/antd/es/button/button.js
+++ b/node_modules/antd/es/button/button.js
@@ -33,7 +33,7 @@ function isString(str) {
   return typeof str === 'string';
 }
 function isUnBorderedButtonType(type) {
-  return type === 'text' || type === 'link';
+  return type === 'text' || type === 'link' || type === 'custom';
 }
 function isReactFragment(node) {
   return /*#__PURE__*/React.isValidElement(node) && node.type === React.Fragment;
